frontend/src/components/Election/Voting/VotePage.tsx

```diff
@@ -44,29 +71,42 @@ const VotePage = ({ election, fetchElection }) => {
     }
     navigate(`/Election/${id}/thanks`)
   }
+
   return (
     <Container disableGutters={true} maxWidth="sm">
-      <BallotSelector
-        race={election.races[currentRace]}
-        candidates={election.races[currentRace].candidates}
-        onUpdate={newRankings => { onUpdate(currentRace, newRankings) }}
-        scores={scores[currentRace]}
+      <BallotPageSelector
+        page={pages[currentPage]}
+        races={election.races}
+        onUpdate={newRankings => { onUpdate(currentPage, newRankings) }}
       />
-      {election.races.length > 1 &&
+      {pages.length > 1 &&
         <Box sx={{ display: 'flex', justifyContent: "space-between" }}>
           <Button
             variant='outlined'
-            onClick={() => setCurrentRace(count => count - 1)}
-            disabled={currentRace === 0}
-            style={{ minWidth:"150px" }}>
-            Previous Race
+            onClick={() => setCurrentPage(count => count - 1)}
+            disabled={currentPage === 0}
+            style={{ minWidth:"150px", marginRight: "40px"}}>
+            Previous Page
           </Button>
+          <Grid xs={pages.length} wrap='nowrap'>
```

Comment:
Should consider using some MUI components to match styles.
https://mui.com/material-ui/react-pagination/
https://mui.com/material-ui/react-stepper/

Reply:
I like how the current system communicates state with the images (📄is an info page, ⬜is a empty ballot, ✅is a completed ballot). I don't see examples for that with the react system but I can dig deeper

I was also thinking of replacing the emojis with images from react-icons

Reply:
The stepper should allow for [custom icons](https://mui.com/material-ui/react-stepper/#customized-horizontal-stepper) and we also have a pretty pretty large icons library already installed https://mui.com/material-ui/material-icons/

Reply:
Sounds good, I update the PR using the stepper function